.gitignore

```diff
@@ -4,3 +4,4 @@
 terraform.tfstate
 terraform.tfvars
 Gemfile.lock
+.idea
```

Comment:
Consider using [global .gitignore](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore) for dirs like `.idea`.